controllers/apply-next/simple/fields.js

```diff
@@ -96,22 +90,19 @@ const allFields = {
             cy: '(WELSH) When is the planned (or estimated) start date of your project?'
         },
         explanation: {
-            en:
-                'This date needs to be at least 12 weeks from when you plan to submit your application. If your project is a one-off event, please tell us the date of the event.',
-            cy:
-                '(WELSH) This date needs to be at least 12 weeks from when you plan to submit your application. If your project is a one-off event, please tell us the date of the event.'
+            en: `
+                <p><strong>For example: 12 11 2020</strong></p>
+                <p>This date needs to be at least 12 weeks from when you plan to submit your application. If your project is a one-off event, please tell us the date of the event.</p>
```

Comment:
I wonder if we should add in the actual (dynamic) date this should be here?

Reply:
Good shout. This relates to the point below, if we configure this on the field we could a) interpolate this into the message and b) workout an appropriate minimum year for the input.